python/ql/src/Security/CWE-078/UnsafeShellCommandConstruction.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Unsafe shell command constructed from library input
+ * @description Using externally controlled strings in a command line may allow a malicious
+ *              user to change the meaning of the command.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 6.3
+ * @precision high
```

Comment:
I think it's a code-smell with high precision, but a security issue with medium precision.  

So I suggest that we lower the severity to `warning` instead of lowering the precision.  
Thoughts?